gr-blocks/include/gnuradio/blocks/rotator_cc.h

```diff
@@ -32,6 +32,18 @@ namespace blocks {
 /*!
  * \brief Complex rotator
  * \ingroup math_operators_blk
+ *
+ * \details
+ *
+ * Message Ports:
+ *
+ * phase_inc (input):
+ *
+ *    Receives a PMT pair with a new phase increment to be set on the rotator at
```

Comment:
Suggestion: Make this a dict. Then the index can be optional. That could actually be useful for applications that don't have an actual control loop.

Reply:
Done